Sources/PrimerSDK/Classes/Extensions & Utilities/UIImageView+Extensions.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIImageView+Extensions.swift
+//  PrimerSDK
+//
+//  Copyright © 2022 Primer API ltd. All rights reserved.
+//
+
+#if canImport(UIKit)
+
+import UIKit
+
+extension UIImageView {
+    
+    func downloaded(from url: URL, contentMode mode: ContentMode = .scaleAspectFit) {
+        contentMode = mode
+        URLSession.shared.dataTask(with: url) { data, response, error in
+            guard
+                let httpURLResponse = response as? HTTPURLResponse, httpURLResponse.statusCode == 200,
+                let mimeType = response?.mimeType, mimeType.hasPrefix("image"),
```

Comment:
Are you sure they're going to return the mime type? Wouldn't the rest of the checks (converting it to a `UIImage`) suffice?

Reply:
Addressed ✅